scripts/eslint/index.js

```diff
@@ -8,12 +8,54 @@
 'use strict';
 
 const CLIEngine = require('eslint').CLIEngine;
-const cli = new CLIEngine();
-const formatter = cli.getFormatter();
+const {npmPath, nodePath, sourcePath} = require('../shared/esPath');
```

Comment:
Let’s rename that file to `filesByLanguageVersion`?

Reply:
ok